spec/spec_helper.rb

```diff
@@ -9,6 +9,8 @@
 require 'shared_examples'
 
 RSpec.configure do |config|
+  config.berkshelf_options = { except: :integration }
```

Comment:
I _think_ it's still needed for the kitchen tests, I'll have to check that though